core/src/test/java/com/linecorp/armeria/client/endpoint/EndpointGroupTest.java

```diff
@@ -34,4 +38,27 @@ public void orElse() throws Exception {
         assertThat(endpointGroup1.orElse(endpointGroup2).endpoints())
                 .isEqualTo(endpointGroup1.endpoints());
     }
+
+    @Test
+    public void updateEndpoints() {
+        EndpointGroup endpointGroup = new EndpointGroup();
+        AtomicInteger updateListenerCalled = new AtomicInteger(0);
```

Comment:
Hm., I want to check here whether callback is called or not. Is Awaitlity also help this case?